### DISCUSSION TITLE
How to modify an unresponsive PR?

### DISCUSSION THREAD

**Discussion by buhtz** [community/community#32123] [Pull Requests] [answered]
Let's assume someone create a PR (against `main`). After then the PR opener isn't available anymore and doesn't answer any questions nor modify his PR if needed.

When the maintainer want to have a modification to that PR before merging. How can he do this?

The PR comes from a fork (`foreign:patch-1`) and should merged into `upstream:main`.

**Comment by airtower-luna** [comment, marked as answer]
> When the maintainer want to have a modification to that PR before merging. How can he do this?

The easiest option is if the person who opened the PR enabled the option that allows people with write access to the target repository to push to the PR branch. In that case the maintainer can just push to the branch in the fork, and merge the PR once they're happy with the result.

If that's not the case, the maintainer can check out the PR branch, push to a new branch in their own repository, and create an new PR (linking to the original one in the description might make sense). See [Modifying an inactive pull request locally](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally#modifying-an-inactive-pull-request-locally) for details.
